web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1707,6 +1707,10 @@ public void executing(ChangeSet changeSet, int numChangeSetsToRun) {
 						// start openmrs
 						try {
 							UpdateFilter.setUpdatesRequired(false);
+							
+							// rebuild lucene search indexes
+							Context.updateSearchIndex();
```

Comment:
@jdegraft do you think this change is necessary? Note that the next call WebDaemon.startOpenmrs will eventually call Context. updateSearchIndex() via Context.startup()